server/templates/base.html

```diff
@@ -13,78 +12,139 @@
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
--->
+#}
+
+{#
+  Required variables:
+  title: page title
+  main_id: DOM Element ID for the main content block
+
+  Blocks to override:
+  head - additional head elements
+  content - main page content
+#}
+
+
+<!DOCTYPE html>
 
 <html lang="en">
 <head>
-  <!-- Google Tag Manager -->
-  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-  })(window,document,'script','dataLayer','GTM-596VBP2');</script>
-  <!-- End Google Tag Manager -->
```

Comment:
Where is the GTM moved? This should be linked to the current GA project already?

Reply:
It won't be replaced, and I'll add back GA in a followup (discussed this in our standup earlier this week that we don't need this anymore since this is mostly for ad tracking).